sql/src/main/java/io/crate/analyze/TablePropertiesAnalyzer.java

```diff
@@ -194,8 +196,9 @@ private static void validateTableProperties(TableParameterInfo tableParameterInf
     protected static class NumberOfReplicasSettingApplier extends SettingsAppliers.AbstractSettingsApplier {
 
         private static final Settings DEFAULT = Settings.builder()
-            .put(TableParameterInfo.NUMBER_OF_REPLICAS, 1)
-            .put(TableParameterInfo.AUTO_EXPAND_REPLICAS, false)
+            .put(TableParameterInfo.NUMBER_OF_REPLICAS, 0)
+            .put(TableParameterInfo.AUTO_EXPAND_REPLICAS, "0-1")
+            .put(TableParameterInfo.SETTING_WAIT_FOR_ACTIVE_SHARDS, "all")
```

Comment:
Could we also use the default values from the original settings definition here to prevent them from getting out of sync?